src/shell.rs

```diff
@@ -451,19 +451,19 @@ fn get_verbosity(
 
 pub trait Stream {
     type TTY: IsTerminal;
-    const OWO: owo_colors::Stream;
+    const OWO: Option<owo_colors::Stream>;
 
     #[must_use]
     fn is_atty() -> bool;
 
-    fn owo(&self) -> owo_colors::Stream {
+    fn owo(&self) -> Option<owo_colors::Stream> {
         Self::OWO
     }
 }
 
 impl Stream for io::Stdin {
     type TTY = io::Stdin;
-    const OWO: owo_colors::Stream = owo_colors::Stream::Stdin;
+    const OWO: Option<owo_colors::Stream> = None;
```

Comment:
Another option is to do `const OWO: owo_colors::Stream = owo_colors::Stream::Stdout;` here even though that is wrong, but it avoids the `Option` and the `unwrap()`.

Or split the trait.  I was a bit worried as it is `pub`.  OTOH, probably nobody else is using the lib.

Reply:
Breaking the api is fine. We dont guarantee that its stable

https://github.com/cross-rs/cross/blob/4090beca3cfffa44371a5bba524de3a578aa46c3/src/lib.rs#L8